### PR TITLE
Forcepoint Email Security Mapping

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3355)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ForcepointEmailSecurity/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Forcepoint Email Security
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Forcepoint Email Security to forward Syslog messages in CEF format.
+
+From the Forcepoint Security Manager, follow these instructions;
+1. Navigate to the page **Settings** > **General** > **SIEM Integration**.
+2. Mark the check box **Enable SIEM integration for all email appliances**
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:8]:
```suggestion
2. Select the checkbox **Enable SIEM integration for all email appliances**.
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+3. Configure the following SIEM settings;
+   3.1. At **IP address or hostname**, enter the relevant IP address or hostname.
+   3.2. At **Port**, enter the relevant Port number.
+   3.3. From the **Transport protocol** section, enter the relevant protocol (**UDP**).
+   3.4. From the **SIEM format** pull-down menu, enter the relevant format (**syslog/CEF**).
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:13]:
```suggestion
   d. From the **SIEM format** dropdown menu, enter the relevant format (**syslog/CEF**).
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Forcepoint Email Security
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Forcepoint Email Security to forward Syslog messages in CEF format.
+
+From the Forcepoint Security Manager, follow these instructions;
+1. Navigate to the page **Settings** > **General** > **SIEM Integration**.
+2. Mark the check box **Enable SIEM integration for all email appliances**
+3. Configure the following SIEM settings;
+   3.1. At **IP address or hostname**, enter the relevant IP address or hostname.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:10]:
```suggestion
   a. At **IP address or hostname**, enter the relevant IP address or hostname.
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+   3.2. At **Port**, enter the relevant Port number.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:11]:
```suggestion
   b. At **Port**, enter the relevant Port number.
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+   3.3. From the **Transport protocol** section, enter the relevant protocol (**UDP**).
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:12]:
```suggestion
   c. From the **Transport protocol** section, enter the relevant protocol (**UDP**).
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+   3.4. From the **SIEM format** pull-down menu, enter the relevant format (**syslog/CEF**).
+   3.5. Confirm configuration by clicking on **Send Test Message**
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:14]:
```suggestion
   e. Confirm configuration by clicking **Send Test Message**.
```

File: Packs/ForcepointEmailSecurity/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+   * Click on **OK** once you are done.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointEmailSecurity/README.md:15]:
```suggestion
4. Click **OK** once you are done.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**